user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+.. _user-guide-user-management-permissions-basic:
+
+Access and Permissions Basics
+=============================
+
+OroCRM provides a lot of functionality that can enhance your CRM process. These may be used by different members of the 
+team. However, it is obvious that information and tools used by a marketing associate are different from 
+those required by support teams, or by the company management. Flexible structure of the access and permissions settings 
+helps to ensure that all the information is safe and available right where it is required, and all the users gain access 
+to the features and capabilities meeting their needs.
```

Comment:
Edit: The functionality built within OroCRM can enhance your CRM process. However, different members of the team will require different information and tools. For instance, the needs of marketing associates are different from those of support teams or company management. Because of this, OroCRM allows a good deal of flexibility when it comes to adjusting access and permission settings. This helps to ensure that all information is safe but still easily available. All users will be able to use the features and capabilities that they require.